pkgs/os-specific/linux/lvm2/common.nix

```diff
@@ -47,9 +54,11 @@ stdenv.mkDerivation rec {
     "ac_cv_func_malloc_0_nonnull=yes"
     "ac_cv_func_realloc_0_nonnull=yes"
   ] ++
-  lib.optionals (udev != null) [
+  lib.optionals udevSupport [
```

Comment:
Should this be on the previous line to match the style below?

Reply:
Done

Reply:
Done